rbac/common/transactions/user_transactions.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+""" Common Transaction Creation
+"""
+import rethinkdb as r
+from rbac.server.db.db_utils import create_connection
+from rbac.common.role.delete_role_owner import DeleteRoleOwner
+from rbac.common.sawtooth import batcher
+
+
+async def create_delete_role_owner_txns(key_pair, next_id, txn_list):
```

Comment:
This exact function is used in LDAP delta inbound sync. Can you use this function during the LDAP delta inbound sync as well?

Reply:
After discussions with Vamshi, these functions won't work over in LDAP delta sync because these are async functions.